python/dask_cudf/dask_cudf/io/json.py

```diff
@@ -6,4 +6,17 @@
 
 import cudf
 
-read_json = partial(dask.dataframe.read_json, engine=cudf.read_json)
+
+def read_json(path, engine="auto", **kwargs):
+    # Wrap `dd.read_json` with special engine handling
```

Comment:
Does it make sense to again add a short docstring here?

Reply:
Yeah - I'll add a new docstring now that the engine logic is more expansive than `dd.read_json`